### PR TITLE
week4 update idoft - xinyinglyu

### PR DESCRIPTION
**Pull request by xinying-7**
Changing status for 28 different lines with evidences written in progress4.md

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -4050,7 +4050,7 @@ https://github.com/fluent/fluent-logger-java,da14ec349bf0904da9865940b57b8756361
 https://github.com/fluent/fluent-logger-java,da14ec349bf0904da9865940b57b87563616ee04,.,org.fluentd.logger.TestFluentLogger.testReconnection,NDOD;UD;NOD,Unmaintained,,last commit on 2019-06-12
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-engine,net.redpipe.engine.swagger.SwaggerApiTest.checkSwagger,NOD,RepoArchived,,
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkMail,NOD,RepoArchived,,
-https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,Opened,https://github.com/fromage/redpipe/pull/56,
+https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,Skipped,https://github.com/FroMage/redpipe/commit/2fc605bc2fad4aadbb3b05282f528a05673df2ca#diff-86139da6bd888a5e7589f89d4f81a426bac99cd8aabab363efb5e338c1158f5bR131,
```
Comment (darko-marinov) [inline pr-data.csv:4053]: Please leave `Opened` as it was and put `RepoArchived` in the last column (`Notes`).

File: pr-data.csv [hunk 2]
```diff
@@ -5451,7 +5451,7 @@ https://github.com/pippo-java/pippo,ae898b6c478af7365b04c2f5bf9331abf496b166,pip
 https://github.com/pippo-java/pippo,ae898b6c478af7365b04c2f5bf9331abf496b166,pippo-session-parent/pippo-session-mongodb,ro.pippo.session.mongodb.MongoDBSessionDataStorageTest.testSave,NOD,,,
 https://github.com/pmd/pmd,bdc08e44fd92380d59cdc50cf9d8dd36e537c369,pmd-core,net.sourceforge.pmd.lang.rule.internal.LatticeRelationTest.testToString,ID,Opened,https://github.com/pmd/pmd/pull/4752,
 https://github.com/pmd/pmd,bdc08e44fd92380d59cdc50cf9d8dd36e537c369,pmd-core,net.sourceforge.pmd.renderers.SummaryHTMLRendererTest.testRendererMultiple,ID,Accepted,https://github.com/pmd/pmd/pull/4750,
-https://github.com/ppiastucki/recast4j,01d478e190b731d2fe861e3b969b4c8dd04f0ede,detourtilecache,org.recast4j.detour.tilecache.TileCacheTest.testPerformance,NOD,,,
+https://github.com/ppiastucki/recast4j,01d478e190b731d2fe861e3b969b4c8dd04f0ede,detourtilecache,org.recast4j.detour.tilecache.TileCacheTest.testPerformance,NOD,MovedToGradle,,
```
Comment (darko-marinov) [inline pr-data.csv:5454]: Add in the last column the commit when the project moved to Gradle.

File: pr-data.csv [hunk 3]
```diff
@@ -5911,16 +5911,16 @@ https://github.com/square/dagger,0fa73e0ee07015f191c25b1f3fdf7048321d1610,core,d
...
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.SingleThrowingTest.bodyThrowingInOnSuccessDeliveredToPlugin,NDOD;NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.SingleThrowingTest.responseThrowingInOnErrorDeliveredToPlugin,NDOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.SingleThrowingTest.responseThrowingInOnSuccessDeliveredToPlugin,NDOD;NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit,retrofit2.CallTest.cancelOkHttpRequest,NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit,retrofit2.CallTest.cancelRequest,NDOD;NOD,,,
+https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.CancelDisposeTest.cancelDoesNotDispose,NOD,MovedToGradle,,
```
Comment (darko-marinov) [inline pr-data.csv:5914]: As above -- add commits when the project moved to Gradle. Did you try to run this project with the NonDex Gradle plugin?

File: pr-data.csv [hunk 4]
```diff
@@ -6626,25 +6626,25 @@ https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,ript
...
-https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.PluginTest.shouldUseBackupRequestPlugin,NOD,,,
-https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.PluginTest.shouldUseOriginalStackTracePlugin,NOD,,,
-https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.retry.RetryTest.shouldRetryForAtMostMaxRetriesTimes,NOD,,,
-https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.url.UrlResolutionTest.shouldAppendUrl,NOD,,,
-https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-backup,org.zalando.riptide.backup.BackupRequestPluginTest.shouldUseBackupRequest,NOD,,,
+https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.CircuitBreakerTest.shouldAutowireCircuitBreakers,NOD,Skipped,,
```
Comment (darko-marinov) [inline pr-data.csv:6629]: You should have `Skipped` here. Please remove (or revise) these changes.

File: pr-data.csv [hunk 5]
```diff
@@ -6626,25 +6626,25 @@ https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,ript
...
+https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-backup,org.zalando.riptide.backup.BackupRequestPluginTest.shouldUseBackupRequest,NOD,Skipped,,
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-failsafe,org.zalando.riptide.failsafe.FailsafePluginCircuitBreakerTest.shouldOpenCircuit,UD,,,
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-failsafe,org.zalando.riptide.failsafe.FailsafePluginRetriesTest.shouldInvokeListenersOnRetryableResult,UD,,,
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-failsafe,org.zalando.riptide.failsafe.FailsafePluginRetriesTest.shouldNotRetryNonIdempotentMethod,UD,,,
-https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-failsafe,org.zalando.riptide.failsafe.RetryAfterDelayFunctionTest.shouldRetryOnDemandWithDynamicDelay,NOD,,,
+https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-failsafe,org.zalando.riptide.failsafe.RetryAfterDelayFunctionTest.shouldRetryOnDemandWithDynamicDelay,NOD,Skipped,,
```
Comment (darko-marinov) [inline pr-data.csv:6647]: As above -- no `Skipped` here in IDoFT.

**Comment by darko-marinov** [comment]
Any progress on splitting this PR into multiple? Please let me know if you need some help.

**Comment by darko-marinov** [comment]
You still need to clean this PR if you want to get some "points".

**Comment by xinying-7** [comment]
please refer to  https://github.com/TestingResearchIllinois/idoft/pull/1597  to see the changes that corrects these issues.

**Comment by darko-marinov** [comment]
Can you simplify this PR to keep only a few changes that I can easily accept? Then we can look into the other changes in #1597

**Comment by xinying-7** [comment]
I've revised this PR and moved all the changes in https://github.com/TestingResearchIllinois/idoft/pull/1597 to here , below is the summary of changes:

Changed status to movedToGradle and added evidence for 8 tests in retrofit-adapters/rxjava, plus 1 test in https://github.com/ppiastucki/recast4j, Evidence/log is in progress_4.md.

Changed the status of riptide: org.zalando.riptide.autoconfigure.retry.RetryTest.shouldRetryForAtMostMaxRetriesTimes to MovedOrRenamed. Evidence/log is in progress_4.md.

Changed the status of riptide: org.zalando.riptide.autoconfigure.LogbookCompatibilityTest.shouldUseInterceptors to Deleted. Evidence/log is in progress_4.md.

Removed all the “skipped” status changes.

[https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,](https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,Opened,https:/github.com/fromage/redpipe/pull/56,)
The RepoArchived status of this one was already been updated by others so didn't change this line.

**Pull request merged**